snapd-glib/snapd-system-information.c

```diff
@@ -219,6 +221,23 @@ snapd_system_information_get_os_version (SnapdSystemInformation *system_informat
     return system_information->os_version;
 }
 
+/**
+ * snapd_system_information_get_sandbox_features:
+ * @system_information: a #SnapdSystemInformation.
+ *
+ * Gets the sandbox features that snapd has.
```

Comment:
"that snapd has"? Maybe "that snapd requires"?

It should also document what the key's value is.